src/main/java/com/onarandombox/MultiverseCore/listeners/MVPlayerListener.java

```diff
@@ -317,6 +322,21 @@ public void playerPortal(PlayerPortalEvent event) {
         }
     }
 
+    private void sendPlayerToJoinLocation(final Player player) {
+        this.plugin.getServer().getScheduler().scheduleSyncDelayedTask(this.plugin,
```

Comment:
`scheduleSync` is badly named, use `runTaskLater`

Reply:
Used it as it was used for other parts of mv code (such as `handleGameModeAndFlight`), will have to discuss with @dumptruckman and @nicegamer7 if the change to `runTaskLater` should be used, then change it for all.,